enterprise-suite/console-operator/deploy/crds/lightbend_v1alpha1_console_cr.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: lightbend.com/v1alpha1
+kind: Console
+metadata:
+  name: example-console
+spec:
+  # Default values copied from <project_dir>/helm-charts/enterprise-suite/values.yaml
+  
+  #################################################
+  # Console UI Configuration
+  #
+  consoleUIConfig:
+    logo: ""
+    isMonitorEditEnabled: false
+  
+  #################################################
+  # Update these versions for a release.
+  #
+  # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-console
+  esConsoleVersion: v1.2.5
+  # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/console-api
+  esMonitorVersion: v1.2.2
+  # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-grafana
+  esGrafanaVersion: v0.3.0
+  # prom/prometheus
+  prometheusVersion: v2.9.2
+  # jimmidyson/configmap-reload
+  configMapReloadVersion: v0.2.2
+  # gcr.io/google_containers/kube-state-metrics
+  kubeStateMetricsVersion: v1.2.0
+  # lightbend/go-dnsmasq
+  goDnsmasqVersion: v0.1.7-1
+  # alpine
+  alpineVersion: "3.8"
+  # busybox
+  busyboxVersion: "1.30"
+  
+  #################################################
+  # Image names to use. Combined with the versions above, gives the fully qualified docker image name.
+  #
+  # es-console image
+  esConsoleImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/es-console'
+  # console-api image
+  esMonitorImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/console-api'
+  # es-grafana image
+  esGrafanaImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/es-grafana'
+  # prometheus image
+  prometheusImage: prom/prometheus
+  # configmap-reload image
+  configMapReloadImage: jimmidyson/configmap-reload
+  # kube-state-metrics image
+  kubeStateMetricsImage: gcr.io/google_containers/kube-state-metrics
+  # go-dnsmasq image
+  goDnsmasqImage: lightbend-docker-registry.bintray.io/lightbend/go-dnsmasq
+  # alpine image
+  alpineImage: alpine
+  # busybox image
+  busyboxImage: busybox
+  
+  #################################################
+  # Kubernetes Api Versions
+  #
+  rbacApiVersion: rbac.authorization.k8s.io/v1
+  apiGroupVersion: rbac.authorization.k8s.io
+  deploymentApiVersion: apps/v1beta2
+  daemonSetApiVersion: apps/v1beta2
+  
+  #################################################
+  # Resource Settings
+  #
+  # default image pull  policy
+  imagePullPolicy: IfNotPresent
+  # prometheus annotation domain, e.g. `domain/scrape=true`
+  prometheusDomain: prometheus.io
+  # Run pods as the given UID. There is no sensible default for this, so customers need to provide it.
+  podUID:
+  # Commercial image credentials - set `username` and `password` to install as Kubernetes Secret.
+  imageCredentials:
+    registry: lightbend-docker-commercial-registry.bintray.io
+  #  username: setme
+  #  password: setme
```

Comment:
This is fairly awkward - users should be setting up the secret directly themselves, otherwise the credentials are exposed in the CR. It is probably okay though for our simple operator.